docs/conversion/sakai_21_mysql_conversion.sql

```diff
@@ -65,3 +65,186 @@ UPDATE rbc_tool_item_rbc_assoc SET created = NOW() WHERE created IS NULL;
+ALTER TABLE POLL_POLL ALTER POLL_IS_PUBLIC DROP DEFAULT;
+ALTER TABLE POLL_POLL MODIFY POLL_LIMIT_VOTE BIT(1) NOT NULL;
+ALTER TABLE POLL_POLL ALTER POLL_LIMIT_VOTE DROP DEFAULT;
+ALTER TABLE POLL_POLL MODIFY POLL_MAX_OPTIONS INT NOT NULL;
+ALTER TABLE POLL_POLL ALTER POLL_MAX_OPTIONS DROP DEFAULT;
+ALTER TABLE POLL_POLL MODIFY POLL_MIN_OPTIONS INT NOT NULL;
+ALTER TABLE POLL_POLL ALTER POLL_MIN_OPTIONS DROP DEFAULT;
+ALTER TABLE POLL_POLL MODIFY POLL_OWNER VARCHAR(255) NOT NULL;
+ALTER TABLE POLL_POLL ALTER POLL_OWNER DROP DEFAULT;
+ALTER TABLE POLL_POLL MODIFY POLL_SITE_ID VARCHAR(255) NOT NULL;
+ALTER TABLE POLL_POLL ALTER POLL_SITE_ID DROP DEFAULT;
+ALTER TABLE POLL_POLL MODIFY POLL_TEXT LONGTEXT NOT NULL;
+ALTER TABLE POLL_POLL ALTER POLL_TEXT DROP DEFAULT;
+ALTER TABLE POLL_POLL MODIFY POLL_UUID VARCHAR(255) NOT NULL;
+ALTER TABLE POLL_POLL ALTER POLL_UUID DROP DEFAULT;
+ALTER TABLE POLL_POLL MODIFY POLL_VOTE_CLOSE datetime NOT NULL;
+ALTER TABLE POLL_POLL ALTER POLL_VOTE_CLOSE DROP DEFAULT;
+ALTER TABLE POLL_POLL MODIFY POLL_VOTE_OPEN datetime NOT NULL;
+ALTER TABLE POLL_POLL ALTER POLL_VOTE_OPEN DROP DEFAULT;
+
+ALTER TABLE POLL_VOTE MODIFY VOTE_DATE datetime NOT NULL;
+ALTER TABLE POLL_VOTE ALTER VOTE_DATE DROP DEFAULT;
+ALTER TABLE POLL_VOTE MODIFY VOTE_IP VARCHAR(255) NOT NULL;
+ALTER TABLE POLL_VOTE ALTER VOTE_IP DROP DEFAULT;
+ALTER TABLE POLL_VOTE MODIFY VOTE_POLL_ID BIGINT NOT NULL;
+ALTER TABLE POLL_VOTE ALTER VOTE_POLL_ID DROP DEFAULT;
+ALTER TABLE POLL_VOTE MODIFY VOTE_SUBMISSION_ID VARCHAR(255) NOT NULL;
+ALTER TABLE POLL_VOTE ALTER VOTE_SUBMISSION_ID DROP DEFAULT;
+ALTER TABLE POLL_VOTE MODIFY USER_ID VARCHAR(255) NOT NULL;
+ALTER TABLE POLL_VOTE ALTER USER_ID DROP DEFAULT;
+
+
+-- SiteStats
+ALTER TABLE SSQ_ANSWER MODIFY ANSWER VARCHAR(255) NOT NULL;
+ALTER TABLE SSQ_ANSWER ALTER ANSWER DROP DEFAULT;
+ALTER TABLE SSQ_ANSWER MODIFY FILL_IN_BLANK BIT(1) NOT NULL;
+ALTER TABLE SSQ_ANSWER ALTER FILL_IN_BLANK DROP DEFAULT;
+ALTER TABLE SSQ_ANSWER MODIFY ORDER_NUM INT NOT NULL;
+ALTER TABLE SSQ_ANSWER ALTER ORDER_NUM DROP DEFAULT;
+
+ALTER TABLE SSQ_QUESTION MODIFY IS_CURRENT VARCHAR(255) NOT NULL;
+ALTER TABLE SSQ_QUESTION ALTER IS_CURRENT DROP DEFAULT;
+ALTER TABLE SSQ_QUESTION MODIFY MULTIPLE_ANSWERS BIT(1) NOT NULL;
+ALTER TABLE SSQ_QUESTION ALTER MULTIPLE_ANSWERS DROP DEFAULT;
+ALTER TABLE SSQ_QUESTION MODIFY ORDER_NUM INT NOT NULL;
+ALTER TABLE SSQ_QUESTION ALTER ORDER_NUM DROP DEFAULT;
+ALTER TABLE SSQ_QUESTION MODIFY REQUIRED BIT(1) NOT NULL;
+ALTER TABLE SSQ_QUESTION ALTER REQUIRED DROP DEFAULT;
+
+ALTER TABLE SSQ_USER_ANSWER MODIFY SITE_ID VARCHAR(255) NOT NULL;
+ALTER TABLE SSQ_USER_ANSWER ALTER SITE_ID DROP DEFAULT;
+ALTER TABLE SSQ_USER_ANSWER MODIFY USER_ID VARCHAR(255) NOT NULL;
+ALTER TABLE SSQ_USER_ANSWER ALTER USER_ID DROP DEFAULT;
+
+ALTER TABLE SSQ_SITETYPE_QUESTIONS MODIFY SITE_TYPE VARCHAR(255) NOT NULL;
+ALTER TABLE SSQ_SITETYPE_QUESTIONS ALTER SITE_TYPE DROP DEFAULT;
```

Comment:
Comment here indicates these are for sitestats, but sitestats tables start with SST not SSQ

Reply:
i thought it was sitestats! its really site setup questions!